tsconfig.base.json

```diff
@@ -7,7 +7,7 @@
     "pretty": true,
     "module": "Node16",
     "moduleResolution": "Node16",
-    "target": "ES2021",
+    "target": "ES2022",
```

Comment:
Updated to this because of [this recommendation ](https://www.npmjs.com/package/@tsconfig/node18)